app/views/reports/index.html.erb

```diff
@@ -1,46 +1,35 @@
-<%= link_to "Back", :back %>
+<h1>Report Categories</h1>
```

Comment:
@Dustin are we missing a link to back out of this page?

Reply:
Otherwise, LGTM

Reply:
@derricklannaman I excluded it from any page that had a direct link in the sidebar.

Screenshot here for reference: <img width="1854" alt="reports" src="https://user-images.githubusercontent.com/7771/94317197-caeb2d00-ff53-11ea-9eeb-a194f4ccdb39.png">

Reply:
Cool